index.html

```diff
@@ -380,6 +383,34 @@ <h1 class="inter700">I'm always interested in hearing about new projects, so if
         </div>
     </footer>
 
+    <!-- menu-mobile-section -->
+    <nav id="mobile-menu" class="mobile-menu">
+        <div id="close-menu-button">
+            <a href="#"><img src="./icons/closeIcon.svg" alt="close menu" /></a>
+        </div>
+        <div class="mobile-menu-list-wrapper">
+            <ul id="mobile-menu-list">
+                <li class="noTagList mobile-menu-li">
+                    <a href="#works" class='mobile-menu-link inter700'>
+                        Portfolio
+                    </a>
+                </li>
+                <li class="noTagList mobile-menu-li">
+                    <a href="#about" class='mobile-menu-link inter700'>
+                        About
+                    </a>
+                </li>
+                <li class="noTagList mobile-menu-li">
+                    <a href="#contact" class='mobile-menu-link inter700'>
+                        Contact
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </nav>
```

Comment:
- [ ] [ optional ] it would be better if the navigation links work on the desktop version also 
<img width="345" alt="Screenshot 2023-04-11 at 23 13 05" src="https://user-images.githubusercontent.com/61790587/231308311-1970192f-9668-4fdc-9275-2c2014cf0ccd.png">